references/cicd-sharedflow-pipeline/test/integration/apiproxy/policies/FC-Security.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<FlowCallout async="false" continueOnError="false" enabled="true" name="FC-Security">
+    <DisplayName>FC-Security</DisplayName>
+    <FaultRules/>
```

Comment:
any empty tags can be removed to make code easier to read

Reply:
Done